engine/env/spark/src/main/java/org/datacleaner/spark/functions/RowProcessingFunction.java

```diff
@@ -69,8 +71,8 @@
  * This class implements two interfaces because it has two (quite similar)
  * styles of usages in the {@link SparkAnalysisRunner}.
  */
-public final class RowProcessingFunction implements
-        Function2<Integer, Iterator<InputRow>, Iterator<Tuple2<String, NamedAnalyzerResult>>>,
+public final class RowProcessingFunction
+        implements Function2<Integer, Iterator<InputRow>, Iterator<Tuple2<String, NamedAnalyzerResult>>>,
         PairFlatMapFunction<Iterator<InputRow>, String, NamedAnalyzerResult> {
```

Comment:
I am using Eclipse Mars on my Mac, which is never than the eclipse version (Juno I think) that was used when creating the formatter file.
I guess it It might be new features of eclipse's formatter. Looking at it on Mac I see the line-wrapping feature (per element type, in this case the 'implements' clause) in the formatter which probably makes the difference:

![screen shot 2015-12-26 at 22 16 56](https://cloud.githubusercontent.com/assets/291450/12007967/69852c92-ac1e-11e5-9314-1cf564e58632.png)